acrobat/blocks/eventwrapper/eventwrapper.js

```diff
@@ -16,7 +16,32 @@ const CONVERSION_START = 'conversion-start';
 // const UPSELL_DIS = 'upsell-displayed';
 const FADE = 'review fade-in';
 
+const changeAccordionAnalytics = (fragment, fragmentName, alloy) => {
+  const allButtons = fragment.querySelectorAll('button');
+  allButtons.forEach((button, index) => {
+    button.removeAttribute('daa-ll');
+    button.addEventListener('click', function() {
+      const text = this.textContent.trim();
+      const ctaNumber = index + 1;
+      const state = this.getAttribute('aria-expanded') === 'true' ? 'Expand' : 'Collapse';
+      alloy(fragmentName, state, ctaNumber, text);
+    });
+  });
+};
+
 export default function init(element) {
+  const faqFrag = element.querySelector("a[href*='faq'], div[data-path*='faq']");
+  if (faqFrag) {
```

Comment:
@TsayAdobe , can you help me ? I tried to write a test for this part of the code that I added, but the code inside `setInterval` isn't executed by the tests. When I call the `init` function from eventwrapper, that is imported inside the tests, `setInterval` is skipped and because of that I can not test `changeAccordionAnalytics` function. Can you please take a look ? Thanks.